components/collective-page/sections/Contribute.js

```diff
@@ -160,123 +161,168 @@ class SectionContribute extends React.PureComponent {
 
     return (
       <Box pt={[4, 5]}>
-        <ContainerSectionContent>
-          <SectionTitle>
-            <FormattedMessage id="CP.Contribute.Title" defaultMessage="Become a contributor" />
-          </SectionTitle>
-        </ContainerSectionContent>
-
-        {hasContribute && (
-          <Box mb={4} data-cy="financial-contributions">
-            <HorizontalScroller getScrollDistance={this.getContributeCardsScrollDistance}>
-              {(ref, Chevrons) => (
-                <div>
-                  <ContainerSectionContent>
-                    <Flex justifyContent="space-between" alignItems="center" mb={3}>
-                      <H3 fontSize="H5" fontWeight="600" color="black.700">
-                        <FormattedMessage id="CP.Contribute.Financial" defaultMessage="Financial contributions" />
-                      </H3>
-                      <Box m={2} flex="0 0 50px">
-                        <Chevrons />
-                      </Box>
-                    </Flex>
-                  </ContainerSectionContent>
+        {status === 'collectiveCreated' && hasContribute ? (
```

Comment:
This ternary looks a bit complex, could we decouple the `status` conditional from the `hasContribute`?

Reply:
Ideally, `collectiveCreated` should not impact the content of the page, it should just impact the banner. What if the user is coming back to the page?